src/Sarif/NotYetGenerated/SarifRewritingVisitor.cs

```diff
@@ -684,7 +726,14 @@ public virtual Run VisitRun(Run node)
                         var value = node.Files[key];
                         if (value != null)
                         {
-                            node.Files[key] = VisitNullChecked(value);
+                            string newKey = key;
+                            node.Files.Remove(key);
+                            value = VisitNullChecked(value, ref newKey);
+
+                            if (newKey != null)
```

Comment:
>newKey != null [](start = 32, length = 14)

if newKey is null, shouldn't we throw? or at least replace the original key/value?